src/main/common/filter.h

```diff
@@ -96,7 +96,7 @@ float biquadFilterApplyDF1(biquadFilter_t *filter, float input);
 float biquadFilterApply(biquadFilter_t *filter, float input);
 float filterGetNotchQ(uint16_t centerFreq, uint16_t cutoff);
 
-void biquadRCFIR2FilterInit(biquadFilter_t *filter, uint16_t f_cut, float dT);
+void biquadRCFIR2FilterInit(biquadFilter_t *filter, float f_cut, float dT);
 
 void fastKalmanInit(fastKalman_t *filter, float q, float r, float p);
```

Comment:
we need to re-introduce the q/r/p variant for this test

Reply:
I will :+1: